ci-operator/config/openshift/openshift-controller-manager/openshift-openshift-controller-manager-release-4.8.yaml

```diff
@@ -3,10 +3,10 @@ base_images:
     name: "4.8"
     namespace: ocp
     tag: base
-  ocp_builder_rhel-7-golang-1.13:
+  ocp_builder_rhel-7-golang-1.14:
```

Comment:
similar issues here.
`base` and `ocp_builder_rhel-8-golang-1.15-openshift-4.7` are used.  The rest of these are not, remove them.

Reply:
updated in current push